src/SUMMARY.md

```diff
@@ -15,5 +15,5 @@
     - [Random Linear Combination](./design/random-linear-combinaion.md)
         - [Full Runnable Code](./design/random-linear-combinaion/full-runnable-code.md)
     - [Recursion](./design/recursion.md)
-    - [State Write Reversion Note 1](./design/state-write-reversion.md)
-    - [State Write Reversion Note 2](./design/state-write-reversion2.md)
+    - [Reversible Write Reversion Note 1](./design/reversible-write-reversion.md)
+    - [Reversible Write Reversion Note 2](./design/reversible-write-reversion2.md)
```

Comment:
I'm thinking perhaps "Reversion Note" is easier to recognize? Tho I believe people can absolutely tell it's about reversion from this title.

Reply:
There are other parts of the documentation that currently use the term "Reversible write reversion" when explaining how reversion is handled, and that seems fine to me.  So I think it's OK to leave this as is.